### ISSUE TITLE
support ext-workspace protocol

### ISSUE THREAD

**Issue by Kommynct** [hyprwm/Hyprland#8895]
### Description

https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/40/

has been merged, would be nice for hyprland to support it that way panels don't have to support hyprland in particular to show workspaces.

**Comment by vaxerski** [comment]
Yeah as I've said before on twitter, this protocol is quite lacking and sucks for anything more than showing 1 2 3 4.

I will probably eventually get to it, but it's not a priority for me.

**Comment by Kommynct** [comment]
Honestly what does the current implementation do other than send numbers to panels? This is a genuine question I have no idea what else it could do.

I use hyprland-autoname-workspaces to give it icons, but tbh i've never even thought of anything else. I also want it because hyprlands waybar implementation sucks and stops updating all the time and never updates on my secondary monitors.

**Comment by izmyname** [comment]
A little bit offtopic.

Doesn't this protocol contradict wayland idea of  'do basic stuff and offload anything else to compositors' ? 

Sorry if this is an incompetent question, but the more a read about stuff, like this protocol, the more often X11 bloat comes to mind.

**Comment by vaxerski** [comment]
no

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
